spec/mailers/user_mailer_spec.rb

```diff
@@ -15,6 +15,14 @@
   # end
 end
 
+shared_examples_for 'contains reservation' do
+  it 'has reservation link' do
+    # body contains link to the reservation
+    expect(@mail.body).to include('<a href=')
```

Comment:
We should be able to pass the reservation ID number into here to check for the actual link. For example, if we're sending an e-mail about reservation one, then we can

```
expect(@mail.body).to include("<a href='http://0.0.0.0:3000/reservations/#{@res.id}'"
```

You might have to change the single quotes to escaped double quotes, depending on how `link_to` renders the links, but that would be a better test (in my opinion).